mappings/net/minecraft/item/AxeItem.mapping

```diff
@@ -9,3 +9,9 @@ CLASS net/minecraft/class_1743 net/minecraft/item/AxeItem
 		ARG 4 settings
 	METHOD method_20269 (Lnet/minecraft/class_1838;Lnet/minecraft/class_1657;)V
 		ARG 1 p
+	METHOD method_34716 getStrippedState (Lnet/minecraft/class_2680;)Ljava/util/Optional;
```

Comment:
Does all stripping go through this? I suggest
```suggestion
	METHOD method_34716 tryStrip (Lnet/minecraft/class_2680;)Ljava/util/Optional;
```

Reply:
It doesn't actually strip any blocks, just gets the stripped state with the same axis property using the stripped block map.

Reply:
:+1: Right. Please mark this one as resolved.